mmaction/models/losses/hvu_loss.py

```diff
@@ -0,0 +1,126 @@
+import torch  # isort: skip
+import torch.nn.functional as F  # isort: skip
+
+from ..registry import LOSSES  # isort: skip
+from .base import BaseWeightedLoss  # isort: skip
+
+
+@LOSSES.register_module()
+class HVULoss(BaseWeightedLoss):
+    """Calculate the BCELoss for HVU.
+
+    Args:
+        categories (list[str]): Names of tag categories, tags are organized in
+            this order. Default: ['action', 'attribute', 'concept', 'event',
+            'object', 'scene'].
+        category_nums (list[int]): Number of tags for each category. Default:
+            [739, 117, 291, 69, 1679, 248].
+        category_loss_weights (list[float]): Loss weights of categories, it
```

Comment:
`List`, as a mutable data type, may cause danger when set as input args. you can use `tuple`